examples/testapp/src/pages/sub-accounts/components/AddAddressButton.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, Button } from '@chakra-ui/react';
+import { createCoinbaseWalletSDK } from '@coinbase/wallet-sdk';
+import React, { useCallback, useState } from 'react';
+
+export function AddAddressButton({ sdk }: { sdk: ReturnType<typeof createCoinbaseWalletSDK> }) {
+  const [subAccount, setSubAccount] = useState<string>();
+
+  const handleAddAddress = useCallback(async () => {
+    if (!sdk) {
+      return;
+    }
+    const provider = sdk.getProvider();
+    const response = (await provider.request({
+      method: 'wallet_addAddress',
+      params: [
+        {
+          chainId: 84532,
```

Comment:
Idea being this address is limited to one chain, correct?

Reply:
Yes the idea was each account is per chain and we wont sync across chains